examples/resources/powermax_snapshot/variables.tf

```diff
@@ -0,0 +1,19 @@
+# variable "username" {
```

Comment:
@doriac11 is this commented out for some reason?

Reply:
Haha yea Just did while I was testing, I will uncomment. 
Thanks!